boards/nrf52dk/Makefile.include

```diff
@@ -15,6 +15,12 @@ export JLINK_PRE_FLASH := erase\nloadfile $(BINDIR)/softdevice.hex
 export JLINK_FLASH_ADDR := 0x1f000
 export LINKER_SCRIPT ?= $(RIOTCPU)/$(CPU)/ldscripts/$(CPU_MODEL)_sd.ld
 endif
+
+export ROM_SIZE = 0x80000
```

Comment:
Not yet, but it can be useful to define slot sizes depending on the ROM size. I can remove it if necessary.